sumpy/expansion/__init__.py

```diff
@@ -442,6 +441,43 @@ def stored_identifiers(self):
         stored_identifiers, _ = self.get_stored_ids_and_unscaled_projection_matrix()
         return stored_identifiers
 
+    @memoize_method
+    def _get_mi_ordering_key(self):
+        """Calculate the multi-index that appears last in the PDE given the pde_dict
+        A degree lexicographic order with the slowest varying index depending on
+        the PDE is used. For two dimensions, this is either deglex or degrevlex.
```

Comment:
- Grammar?
- Explain this a bit (in a comment)?
- What's the impact?

Reply:
I explained a bit more in 827f067. (I also removed the for two dimensions line as it was unnecessary)